content/300-guides/300-upgrade-guides/800-upgrade-from-prisma-1/02-schema-incompatibilities.mdx

```diff
@@ -386,7 +386,7 @@ model User {
 
 model Profile {
   id     String @id
-  userId String
+  userId String @unique
   User   User   @relation(fields: [userId], references: [id])
 }
 ```
```

Comment:
I am not sure this whole block makes sense any more. The original introspected schema is a 1:n, without the `@unique`. Here the description tells you to just remove `[]` to fix one of the sides - but that is not correct as you _also_ need to add the `@unique` manually now. If you do that, the database is still not matching that and a next migration would add unique or a next introspection would remove the @unique again and produce an invalid schema.

Reply:
Yeah, I agree. We list two workarounds for the situation. The first one for me is rather a solution to make it correct (add the unique). This second one is a hack that fundamentally leaves you in an incorrect state, I don't think we should recommend that.

Reply:
Thanks, I've removed this example